first commit

MoreJagruti opened the pull request.

MoreJagruti committed:
first commit

iamabhiD commented:
Please provide commit message for any commits

iamabhiD reviewed (changes requested):
Please mention commit message for any commits

MoreJagruti committed:
Category page

this commit contain documentation of category page. What it contains is specified here.

MoreJagruti committed:
Category Page Documentation

This commit contain the documentation of category page.
